src/store/index.ts

```diff
@@ -0,0 +1,72 @@
+import { createStore, createLogger } from "vuex";
+import { Dapp, Account, DappLocalState } from "@/types";
+import api from "@/api/dapps";
+
+const debug = process.env.NODE_ENV !== "production";
+
+export default createStore({
+	state: {
+		hasAlgoSigner: false,
+		dapps: [] as Dapp[],
+		userAccounts: [] as Account[],
```

Comment:
why this is a list? I think wallet will always return only one address at a given time.